Add inline documentation (#90)
nethesis/nethvoice-report: ui/src/doc-inline/en/data_session_1_table_session.md

# By Session

> Data broken down *by Session* of the agents selected in the queues
> selected in the set period, the data is grouped according to
> the choice made in the filters.

Reviewer: ```suggestion
Data broken down *by Session* of the agents selected in the queues
during configured period. Data are grouped according to chosen filter criteria.
```

The data *by Session* shows the work activity of login, logout
and pause for each selected agent.

The items shown for the *by Session* data are:

- `Queue Number`
- `Queue Name`
- `Agent`
- `Action`: type of activity performed for example login or pause
- `Start`: date and time of the start of the activity
- `End`: date and time of the end of the activity
- `Duration`: duration of the activity
- `Reason`: if the reason for the activity is specified

If no agent has been selected, all data will be shown
agents, if no queues have been selected, the

> agents of all available queues

Reviewer: ??
